documentation/catalog/guides/quickstart.md

```diff
@@ -70,7 +32,7 @@ different environment than the PySpark portion of the project, and so have their
 own dependency requirements.
 
 For instructions geared specifically towards production deployments, see
-[DEPLOYMENT.md](https://github.com/WordPress/openverse/blob/main/catalog/DEPLOYMENT.md)
```

Comment:
Since you've moved the deployment file to the documentation folder, `DEPLOYMENT.md` should also be removed from the repo!